### DISCUSSION TITLE
Upptime Backstage Plugin

### DISCUSSION THREAD

**Discussion by ScottGuymer** [upptime/upptime#966] [Show and tell]
I recently published a fairly simple backstage plugin I created for displaying upptime details within backstage.

You can take a look here and let me know what you think

https://github.com/philips-software/philips-backstage-plugins/tree/main/workspaces/upptime/plugins/upptime-frontend

**Comment by AnandChowdhary** [comment]
Oh wow, this looks great! Do you have an instance running somewhere or a dashboard screenshot?

**Reply by ScottGuymer** [reply]
I dont have a live demo but it will look something like this on your component page

![image](https://github.com/upptime/upptime/assets/956825/c9ec2b61-fd49-411e-9cf8-af22122a1ac5)

**Reply by ScottGuymer** [reply]
Its pretty simple at the moment but I would be very happy to accept PRs to add more functionality from the upptime status page.
